### DISCUSSION TITLE
Menu display in Table it problem

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2280] [Q&A]
Menu display in Table it problem
i use ScrollArea cover Table. but it problem not display menu

[CodeSanbox](https://codesandbox.io/s/runtime-worker-5iyiii?file=/src/App.js)

**Comment by rtivital** [comment]
https://codesandbox.io/s/vibrant-sound-dnsvxe?file=/src/App.js

**Reply by goodlythink** [reply]
thank you
